controllers/user.controller.js

```diff
@@ -22,101 +21,93 @@ exports.create = (req, res) => {
     email: req.body.email,
     password: hash,
   };
-
-  User.findAll({where: {email: user.email}}).then(exist => {
-    if(exist.length > 0){
-        console.log(exist)
-        return res.status(422).send({
-            message: 'Email has already been taken.'
-        })
-    }else{
-        User.create(user)
-        .then((data) => {
-          res.send(data);
-        })
-        .catch((err) => {
-          res.status(500).send({
-            message: err.message || "Some error occurred while creating new user.",
-          });
-        });
+  try {
+    const userUniqueEmailCheck = await User.findAll({
+      where: { email: user.email },
```

Comment:
if you already have unique contraint, you dont need to check that... but you can if you want :)

Reply:
I did that, bcs I wanted to send more specific error message then `Validation Error`. I couldn't find a better way then checking it again.